pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -270,7 +270,7 @@ public static Properties loadPropertyFiles(String... names) {
 
   public static void initDriver() {
     synchronized (TestUtil.class) {
```

Comment:
Do synchronization on public class type may lead to deadlock in multithread environment. It's better to use private local object to sync on.